src/utils/linters.js

```diff
@@ -48,6 +48,10 @@ function isLinterConfigFromAFullyQualifiedModuleName(specifier, prefix) {
 }
 
 function resolvePresetPackage(flavour, preset, rootDir) {
+  if (flavour === 'tslint') {
+    return preset;
+  }
+
   // inspired from https://github.com/eslint/eslint/blob/5b4a94e26d0ef247fe222dacab5749805d9780dd/lib/config/config-file.js#L347
```

Comment:
Do we have a new reference?

Reply:
Worth to mention the TSLint resolution code as a comment.

Reply:
Sure, just the main docs specify it, it hasn't changed we just always implemented it incorrectly.

I'm still not sure this is the best thing to do though, it feels a bit hacky having a tslint-specific short circuit here. will have another think in case i have a better idea

Reply:
Oh, I point to the line about `// inspired from https://`, not your fast return.

BTW, I suppose TSLint still support preset with syntax `tslint:latest`.

Reply:
Yes I know, I just meant in addition to that any suggested alternatives would be nice :)

As for what they support, yes, `tslint:recommended` and `latest` are too. But we already do this wrong, we only check for `recommended` in both es and tslint (we should just check `{flavour}:*` i guess

Reply:
See the example here: https://palantir.github.io/tslint/usage/configuration/#configuration-presets

Reply:
I see. I think divorce the logic between ESLint and TSLint is the correct way.